torch/ao/sparsity/sparsifier/utils.py

```diff
@@ -36,3 +36,9 @@ def __init__(self, mask):
     def forward(self, x):
         assert self.mask.shape == x.shape
         return self.mask * x
+
+    def state_dict(self, *args, **kwargs):
+        # We don't want to let the parametrizations to save the mask.
```

Comment:
Since we are parametrizing the linear weights, why do we want to not store the mask? Shouldn't this be visible in the linear module state_dict as well?

Reply:
That was one of the design choices that we trhough made sense -- the linear layer uses parameterizations, which are created (and managed) by the sparsifier. When the sparsifier is detached (squashed_mask) the parameterization and the mask is removed from the linear layer.

Given the above, if store the linear layer using the `state_dict` it will store the parametrizations as well. However, the ownership belongs to Sparsifier (by our definitions), which also stores the mask when `state_dict` is dumped. To avoid duplication, we decided to not let linear layer serioalize the fakesparse parameterization.